### ISSUE TITLE
Site Index 

### ISSUE THREAD

**Issue by KimPatch** [DukeLivingHistory/RLH-Site-Code#150]
We want to generally make the interviews more discoverable on the site. We want the user to have some kind of birdseye view where  they can see a bit of information about many interviews at once. We are looking for a relatively simple way to do this.
A couple of questions:
What would it take to add 

1. A thumbnail/tile view to the interview page that shows just the picture with name and interview date below it so several columns and rows of tiles could fit on the page?

2. An index page/Site map with just names and interview dates to give readers a better map of the whole collection. I'm picturing the collections as headings, and the interview names appearing alphabetically under the headings. The last heading would be "Interviews that don't appear in a collection". Collections and interviews would be clickable:
**Collection Name**
  Interview name,  Interview date 
  Interview name,  Interview date
  Timeline name
**Collection Name**
  Interview name,  Interview date 
  Interview name,  Interview date
…
**Interviews that don't appear in a collection** 
  Interview name,  Interview date 
  Interview name,  Interview date
etc.
It would be best if there are several columns so readers have a good birdseye view of collections and interviews even when there are several hundred interviews on the site.

3. The index page described above with a toggle that adds a thumbnail?

**Comment by KimPatch** [comment]
@ethanbutler 
If you can do an off-the-top-of-your-head – or at least a relatively quick estimate – of he above that would be helpful in figuring out priorities.

**Comment by KimPatch** [comment]
@ethanbutler 

Thanks – the spacing looks better and it’s good that the 4 columns let us get more interviews on page. 

Trivial change for clarity:  If you can rename the new field “Index Title” rather than “Subtitle” on the backend I think it will be clearer.

1- I think we can solve several problems by putting the text, which should be just 

Name
Index title 
Date of interview 

below the boxes, similar to the examples that Phil pointed to in his previous message.

It will be less busy, nothing will obscure the picture, and there will be no need to indicate which side to put the box or hover to see the information. 

Note that we don’t need the longer descriptions but we do need the date of the interview.

2- The filtering tools that were on the old interviews page should be brought back as they were before *without* the category grouping and headers (name, reverse name, date interviewed, reverse date, date published, reverse date ). This will let viewers simply look up an interview by name or date.

3- I was picturing the category grouping as an additional sort category on the drop-down (choose this sort option and the interviews would group by category and within each category would be in alphabetical order). I’m assuming this is relatively easy to do but I wasn’t sure and so asked the question below. If you have to leave something out this is the one to leave out – #1 and 2 are more important.  

4- This one is just a question: 
In addition to the filtering tools we originally had a ‘change view” icon, and two different views on this page. 
![image](https://user-images.githubusercontent.com/7329361/43045369-0fcc684a-8d86-11e8-8f9f-e2c804027904.png)

I think it might be useful to preserve this as well but have the new tiled view as default with the tiled icon, and use the list icon con to give the viewer the option to toggle to original view with pictures and longer descriptions (which is currently under the tiled icon). 
**Question:**  how difficult would it be to do this?

**Comment by KimPatch** [comment]
Adding key bits from email thread:

**Question:** What's the greatest number of columns we can have now given the new labels? It seems like six would be fit comfortably. And looking at my 24-inch screen it seems like it might work well to scale up to eight.

The default view should be alphabetical. The filtering tools are already built and functional – they allow the user to toggle between alphabetical (default), date published and date of interview, and reverse. These are all important for finding interviews and giving viewers a good mental map of the Rutherfurd collection. 

**Question:** how much extra work is adding sorting by category to the filtering function (#3 in my previous message above) It should be left out if it's extra work, but if it's trivial to add I think it adds to the mental map of the whole collection. Note that the collection groupings should not show up in the default view either way.

**Comment by KimPatch** [comment]
- [ ] I think there was a misunderstanding about the text box. Fixing this will also sidestep the problem that limits the number of boxes across. The text should appear **below** each box rather than on it.

**Name**
Index title
Date of interview 

below the boxes, similar to the examples that Phil pointed to in his previous message. 

(or it may fit better this way: 
Name, date of interview
Index title)
This was also what Phil was referencing in the comment below when he said “can we see with labels below”

The hover is removed  – great. Understood that it was screen reader friendly and I'm glad you're thinking about that. It's the cognitive load I was thinking of.

Sorting looks great. Understood about the interview date sort.

- [ ]  For the sort by category we need category headings like you had before just on this view. I would recommend making the headings font a bit smaller than you had before.

**Comment by KimPatch** [comment]
@ethanbutler 
There's something wrong with row spacing.  On Chrome/Windows 7 I only see five interviews. On Firefox/Windows 7 I see all 18 interviews. It looks like Phil is having the same issue on his Mac

A couple of tweaks for the text:
1. The name and title are a bit crowded. Can you make the space between them match the space between the title and date?
2. There are three different font sizes and three different colors, which increases busyness and makes it hard to parse.
I think the best way to reduce this and also make the three pieces of information easily distinguishable is to
- make the date color the same blue as the title so there are two different colors rather than three.
- make the date size the same as the second line – it will still be a bit smaller because the second line is all caps– there's plenty of room and it's hard to read as it is.

**Comment by ethanbutler** [comment]
@KimPatch – the style updates are made. I'll have some time to add the collections sorting over the weekend.

**Comment by KimPatch** [comment]
Formatting looks good! Looking forward to seeing the collection sorting.

**Comment by KimPatch** [comment]
Looks good. 
I checked to make sure collections don't show up when all interviews are hidden and as expected they don't -- great. 
I also checked this on an ipad -- looks good. 

4 details: 

1. On large screens (and if you zoom out in smaller screens) there ends up being a lot of space on the sides.  How difficult and how advisable is it to have these wrap at 6 across instead of 4 across instead of having wide margins show up?
![image](https://user-images.githubusercontent.com/7329361/43713746-b25b652a-9948-11e8-8c51-9c8d4d4612a3.png)

2. I'm seeing some artifacts on Chrome/PC -- see the lines in the above picture

3. There's too much space between collections (on Firefox and Chrome/PC). Can you reduce this?
![image](https://user-images.githubusercontent.com/7329361/43712880-b46cd068-9945-11e8-9e24-e3d7b08317d7.png)

4. The sorting dropdown doesn't show up on an iphone, so the only choice here is to scroll through alphabetically.  
**Question:**  is this an oversight or are there reasons for leaving out this functionality on the smartphone view?

**Comment by KimPatch** [comment]
@ethanbutler 
Thanks for the email heads up – checking each of these. 

- [ ] 1. I'm seeing six columns in  Firefox but the margins don't line up with the drop-down or the collection headings.
![image](https://user-images.githubusercontent.com/7329361/44033918-ce3717a8-9ed9-11e8-97aa-4c840d655ae7.png)

- [ ] 2. The Chrome artifacts are gone, but most of the interviews now don't show up.  I'm only seeing four.  These are too few interviews to see if six columns works.

3. Spacing now looks good on Firefox and Chrome – thanks!

- [ ] 3b. This might've been overlooked because it was in email – I suggested
"Interviews Not in a Collection" as an improvement on 3b. wording above. It's confusing to have no label.

4. I'm seeing the sorting drop-down on the iPhone – looks great – thanks.

**Comment by KimPatch** [comment]
@ethanbutler better 3b wording from Katie "Standalone Interviews"

**Comment by KimPatch** [comment]
@ethanbutler 
- [ ] 1. Is good at level of the title but there's still a spacing issue at the very top
![image](https://user-images.githubusercontent.com/7329361/44067345-08dd55b6-9f43-11e8-957a-d082d675125e.png)
2. Looks good 
3b. Looks good – thanks

**Comment by KimPatch** [comment]
Looks good!
